infra/helper.py

```diff
@@ -419,6 +450,113 @@ def check_build(args):
   return exit_code
 
 
+def _get_fuzz_targets(project_name):
+  """Return names of fuzz targest build in the project's /out directory."""
+  command = [
+      'find',
+      _get_output_dir(project_name),
+      '-maxdepth',
+      '1',
+      '-type',
+      'f',
+      '-executable'
+  ]
+  output = subprocess.check_output(command)
+  return [os.path.basename(path) for path in output.split()]
+
+
+def _get_latest_corpus(project_name, fuzz_target, corpus_dir):
+  """Download the latest corpus for the given fuzz target."""
+  corpus_dir = os.path.join(corpus_dir, fuzz_target)
+  if not os.path.exists(corpus_dir):
+    os.makedirs(corpus_dir)
+
+  gcs_url = 'gs://%s-backup.clusterfuzz-external.appspot.com' % project_name
```

Comment:
nit: put in a global

Reply:
Done! Also noticed that I'm using wrong URL when I sync non-backup corpus. Will fix it as well.